src/main/java/teammates/common/util/Const.java

```diff
@@ -872,12 +872,11 @@
         public static final String INSTRUCTOR_DETAILS_LENGTH_INVALID = "Instructor Details must have %d columns";
         
         public static final String LOADING = "<img src=\"/images/ajax-loader.gif\" /><br />";
-        public static final String STUDENT_FIRST_TIME = "<div style=\"text-align:left;\">Welcome stranger :-) "
-                + "<br/><br/>It seems you are not a registered user of TEAMMATES. To use TEAMMATES, a course instructor has to add you to a course first. "
-                + "After that, TEAMMATES will send you an email containing the link to 'join' that course. "
-                + "<br/><br/>Not a stranger to TEAMMATES? Could log in before, but not any more? These are the possible reasons:"
-                + "<br/>1. You used a different Google account to access TEAMMATES in the past. "
-                + "In that case, you need to use the same Google account to access TEAMMATES again. Logout and re-login using the other Google account. "
+        public static final String STUDENT_FIRST_TIME = "<p align='center'><font  color='red'><b>Oops! Your Google account is not known to TEAMMATES.<b></font></p><br><div style=\"text-align:left;\">"
+                + "<br/>To access a course on TEAMMATES, first you need to wait till an instructor adds you to that course and TEAMMATES sends you instructions on how to access that particular course in TEAMMATES."+
+                "<br/><br/>If you 'joined' the course in TEAMMATES using a Google account before, but cannot login anymore, these are the possible reasons:"
+                + "<br/>1. You used a different Google account to access TEAMMATES in the past "
+                + "In that case, you need to use the same Google account to access TEAMMATES again. Logout and re-login using the other Google account."
```

Comment:
Line wrap in a way that a code line doesn't exceed 110 chars (to avoid horizontal scrolling in the GitHub diff view)